doc/v2/documentation_data_format.rst

```diff
@@ -14,187 +15,375 @@ This document explains the PEtab data format.
+
+   * These updates follow language specific rules of which variables are
+     *atomic* or *derived*/*algebraic*. For example, SBML treats species
+     concentrations to be *derived*, meaning they are recomputed based on
+     (updated) species amounts and corresponding compartment sizes.
+
+4. **Events and finalization**
+
+   At this stage, all remaining state- or time-dependent changes encoded in
+   the model are applied, including SBML events. The model state used for the
+   evaluation of trigger functions or the execution of any event assignments
+   (including events with delays) is the model state after (3).
+
+   The initial model state for the new period at time = ``time``
+   (:ref:`v2_experiments_table`) is the model state after all these updates
+   have been applied.
+
+5. **Evaluation of observables**
+
+   If measurements exist for the current timepoint, the observables are
+   evaluated after all changes have been applied. The resulting values are
+   then compared against the corresponding measurements in the measurement table.
+
+.. _v2_experiments_table:
+
+Experiments table
+-----------------
+
+The optional experiments table defines a sequence of experimental
+conditions (i.e., discrete changes; see :ref:`v2_conditions_table`) applied
+to the model.
+
+The experiments table is provided as a tab-separated values (TSV) file with
+the following structure:
+
++---------------------+-------------------+-----------------+
+| experimentId        | time              | conditionId     |
++=====================+===================+=================+
+| PETAB_ID            | NUMERIC or '-inf' | CONDITION_ID    |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                 0 | condition_1     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                10 | condition_2     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |               250 | condition_3     |
++---------------------+-------------------+-----------------+
+| patient_3           |              -inf | condition_1     |
++---------------------+-------------------+-----------------+
+| patient_3           |                 0 | condition_2     |
++---------------------+-------------------+-----------------+
+| intervention_effect |               -20 | no_lockdown     |
++---------------------+-------------------+-----------------+
+| intervention_effect |                20 | mild_lockdown   |
++---------------------+-------------------+-----------------+
+| intervention_effect |                40 | severe_lockdown |
++---------------------+-------------------+-----------------+
+
+The order of the rows is arbitrary, but specifying the rows in ascending order
+of time may improve human readability.
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The time courses table has three mandatory columns ``experimentId``,
```

Comment:
```suggestion
The experiment table has three mandatory columns ``experimentId``,
```

Reply:
experiment*s*

Reply:
"Experiment table" in the figure, but fine for me

Reply:
I think that is inconsistent throughout the documentation, library, ...
Let's address that separately and get this monster merged and out of the way.